### DISCUSSION TITLE
what is the reference to “splitrule="auc"” for function rfsrc？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#181] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/5112397/149659604-aa2f1d9e-d589-40a9-95d1-66c688c8f828.png)

**Comment by ishwaran** [comment, marked as answer]
Currently we do not have a vignette to reference this.  This is something we devised on our own.

**Reply by A-Pai** [reply]
If the user wants to understand and learn this usage，what to do？
